packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -954,15 +967,35 @@ const messages = {
     close: 'Lukk faktaboks',
   },
   myNdla: {
-    resources: 'Ressurser',
-    folders: 'Mapper',
+    resources: '{{count}} Ressurs',
+    resources_plural: '{{count}} Ressurser',
+    folders: '{{count}} Mappe',
+    folders_plural: '{{count}} Mapper',
```

Comment:
Norsk rettskriving seier vel små bokstaver her.
```suggestion
    resources: '{{count}} ressurs',
    resources_plural: '{{count}} ressurser',
    folders: '{{count}} mappe',
    folders_plural: '{{count}} mapper',
```